### PR TITLE
feat: tg bot integration manual added

### PR DESCRIPTION
**Pull request by siandreev**
Added a tutorial about integrating the TonConnect to a telegram bot

## Why is it important?

There are many examples of integrating Ton Connect into frontend applications ([ton-connect/demo-dapp](https://github.com/ton-connect/demo-dapp), [ton-connect/demo-dapp-with-react-ui](https://github.com/ton-connect/demo-dapp-with-react-ui)), as well as many examples of authorization on the backend through tone proof ([ton-connect/demo-dapp-backend](https://github.com/ton-connect/demo-dapp-backend)). However, there are no examples of working with Ton Connect for telegram bots (that is, examples when user sessions are stored and processed on the backend). When integrating the Ton Connect into a telegram bot, there are some difficulties: you need to manage user sessions and close unused connections to wallets.

## Related Issue

https://github.com/ton-society/ton-footsteps/issues/176

### CODE REVIEW

**Comment by siandreev** [comment]
@SwiftAdviser Could you take a took?

**Comment by reveloper** [comment]
@siandreev, Thank you for your contribution! 

I would like to ask you to split the code snippets. It would be better to separate the snippets or hide them in spoilers(use it for inseparable cases as an exception). 

The reason is that each code snippet you place on the page should fit within a regular desktop monitor. To make it easier for readers to follow along, it would be ideal if each portion of the code fits onto one page. Additionally, it would be even better if related comments could also fit onto that same page with their corresponding code.

File: docs/develop/dapps/ton-connect/tg-bot-integration.md [hunk 1]
```diff
@@ -0,0 +1,1391 @@
...
+
+Third screen:
+<Selected wallet QR>
+<Back (opens second screen)>
+<Open selected wallet link> 
+```
+
+Let's start with adding inline keyboard to the `/connect` command handler in the `main.ts`
+```ts
+// src/connect-wallet-menu.ts
```
Comment (LevZed) [inline docs/develop/dapps/ton-connect/tg-bot-integration.md:342]:
```suggestion
// src/main.ts
```

File: docs/develop/dapps/ton-connect/tg-bot-integration.md [hunk 2]
```diff
@@ -0,0 +1,1391 @@
...
+### Add Choose a Wallet button handler
+Now we will add `callback_query` handler. Create a file `src/connect-wallet-menu.ts`:
+
+```ts
+// src/connect-wallet-menu.ts
+import { CallbackQuery } from 'node-telegram-bot-api';
+import { getWallets } from './ton-connect/wallets';
+import { bot } from './bot';
+
+export const callbacks = { // Define buttons callbacks
```
Comment (LevZed) [inline docs/develop/dapps/ton-connect/tg-bot-integration.md:395]:
```suggestion
export const walletMenuCallbacks = { // Define buttons callbacks
```

File: docs/develop/dapps/ton-connect/tg-bot-integration.md [hunk 3]
```diff
@@ -0,0 +1,1391 @@
...
+    } catch {
+        return;
+    }
+
+    if (!callbacks[request.method as keyof typeof callbacks]) {
+        return;
+    }
+
+    callbacks[request.method as keyof typeof callbacks](query, request.data);
+});
```
Comment (LevZed) [inline docs/develop/dapps/ton-connect/tg-bot-integration.md:417]:
```suggestion
    if (!walletMenuCallbacks[request.method as keyof typeof walletMenuCallbacks]) {
        return;
    }

    walletMenuCallbacks[request.method as keyof typeof walletMenuCallbacks](query, request.data);
});
```

File: docs/develop/dapps/ton-connect/tg-bot-integration.md [hunk 4]
```diff
@@ -0,0 +1,1391 @@
...
+```
+
+Editing message media (QR image) is little tricky. We need to store image to the file and send it to the Telegram server. Then we can remove this file.
+See details in the `editQR` function.
+
+In `onOpenUniversalQRClick` handler we just regenerate a QR and deeplink and modify the message.
+
+In `onWalletClick` handler we are creating special QR and universal link for selected wallet only, and modify the message.
+
+Now we have to register this functions as callbacks:
```
Comment (LevZed) [inline docs/develop/dapps/ton-connect/tg-bot-integration.md:584]:
```suggestion
Now we have to register this functions as callbacks (`walletMenuCallbacks`): 
```

File: docs/develop/dapps/ton-connect/tg-bot-integration.md [hunk 5]
```diff
@@ -0,0 +1,1391 @@
...
+```ts
+// src/connect-wallet-menu.ts
+import TelegramBot, { CallbackQuery } from 'node-telegram-bot-api';
+import { getWallets } from './ton-connect/wallets';
+import { bot } from './bot';
+import * as fs from 'fs';
+import { getConnector } from './ton-connect/connector';
+import QRCode from 'qrcode';
+
+export const callbacks = {
```
Comment (LevZed) [inline docs/develop/dapps/ton-connect/tg-bot-integration.md:595]:
```suggestion
export const walletMenuCallbacks = {
```

File: docs/develop/dapps/ton-connect/tg-bot-integration.md [hunk 6]
```diff
@@ -0,0 +1,1391 @@
...
+MANIFEST_URL=https://ton-connect.github.io/demo-dapp-with-react-ui/tonconnect-manifest.json
+WALLETS_LIST_CAHCE_TTL_MS=86400000
+DELETE_SEND_TX_MESSAGE_TIMEOUT_MS=600000
+CONNECTOR_TTL_MS=600000
+```
+
+Now let's use it in the `handelConnectCommand`
+
+```ts
+// src/main.ts
```
Comment (LevZed) [inline docs/develop/dapps/ton-connect/tg-bot-integration.md:1086]:
```suggestion
// src/commands-handlers.ts
```

**Review by LevZed — COMMENTED** [review]
cool tutorial

**Pull request merged**